modules/snooker/src/main/ts/ephox/snooker/api/Warehouse.ts

```diff
@@ -100,11 +123,28 @@ const justCells = function (warehouse: Warehouse) {
   return Arr.flatten(rows);
 };
 
+const justColumns = (warehouse: Warehouse) =>
+  Arr.map(warehouse.columns, (group) => group);
+
+const hasColumns = (warehouse: Warehouse) =>
+  warehouse.columns.length > 0;
+
+const getColumnAtIndex = (warehouse: Warehouse, columnIndex: number) => {
+  const result = Arr.find(warehouse.columns, (column): boolean =>
```

Comment:
This would be better if you made `columns` a Record and indexed based on the column "key"/number similar to `access`. It'd also be more performant, as it's O(1) instead of O(n) to do the lookup.